gatsby-node.js

```diff
@@ -46,7 +47,7 @@ exports.onCreateNode = ({ actions, getNode, node }) => {
     createNodeField({ node, name: 'slug', value: slug });
 
     if (pageType === 'releases') {
-      const [_, releaseVersion] = slugParts;
+      const [, releaseVersion] = slugParts;
```

Comment:
eslint was yelling about the `_` variable name. Rather than ignore the rule, I deleted the unused variable.